examples/example_004.py

```diff
@@ -0,0 +1,166 @@
+    results = np.zeros([time.size, 12])
+
+    results[0, 0:3] = lin_vel
+    results[0, 3:6] = ang_vel
+    results[0, 6:9] = theta, phi, psi_0
+    results[0, 9:12] = x_0, y_0, h
+    alpha = np.empty_like(time)
+    alpha[0] = alpha_
+    beta = np.empty_like(time)
+    beta[0] = beta_
+    TAS = np.empty_like(time)
+    TAS[0] = TAS_
+
+    # Linear Momentum and Angular Momentum eqs.
+    equations = System(integrator='dopri5',
+                       model='euler_flat_earth',
+                       jac=False)
+    u, v, w = lin_vel
+    p, q, r = ang_vel
+
+    equations.set_initial_values(u, v, w,
+                                 p, q, r,
+                                 theta, phi, psi_0,
+                                 x_0, y_0, h)
+
+    _, _, rho, _ = atm(h)
+
+    # Define control vectors.
+    delta_e, delta_ail, delta_r, delta_t = control_vector
+    d_e = np.ones_like(time) * delta_e
+    d_e[np.where(time<2)] = delta_e * 1.30
+    d_e[np.where(time<1)] = delta_e * 0.70
+    d_a = np.ones_like(time) * delta_ail
+    d_r = np.ones_like(time) * delta_r
+    d_t = np.ones_like(time) * delta_t
+
+    attitude = theta, phi, psi_0
+
+    # Rename function to make it shorter
+    forces_and_moments = cessna_310.get_forces_and_moments
+    for ii, t in enumerate(time[1:]):
+
+        forces, moments = forces_and_moments(TAS[ii], rho, alpha[ii], beta[ii],
+                                             d_e[ii], 0, d_a[ii], d_r[ii], d_t[ii],
+                                             attitude)
+
+        results[ii+1, :] = equations.propagate(mass, inertia, forces, moments,
+                                               dt)
+
+        lin_vel = results[ii+1, 0:3]
+        ang_vel = results[ii+1, 3:6]
+        attitude = results[ii+1, 6:9]
+        position = results[ii+1, 9:12]
+
+        alpha[ii+1], beta[ii+1], TAS[ii+1] = calculate_alpha_beta_TAS(*lin_vel)
+
+        _, _, rho, _ = atm(position[2])
+
+
+    velocities = results[:, 0:6]
+    attitude_angles = results[:, 6:9]
+    position = results[:, 9:12]
+
+    # PLOTS
+    plt.close('all')
+    plt.style.use('ggplot')
+
+    plt.figure('pos')
+    plt.plot(time, position[:, 0], label='x')
+    plt.plot(time, position[:, 1], label='y')
+    plt.plot(time, position[:, 2], label='z')
+    plt.xlabel('time (s)')
+    plt.ylabel('position (m)')
+    plt.legend()
+
+    plt.figure('angles')
+    plt.plot(time, attitude_angles[:, 0], label='theta')
+    plt.plot(time, attitude_angles[:, 1], label='phi')
+    plt.plot(time, attitude_angles[:, 2], label='psi')
+    plt.xlabel('time (s)')
+    plt.ylabel('attitude (rad)')
+    plt.legend()
+
+    plt.figure('velocities')
+    plt.plot(time, velocities[:, 0], label='u')
+    plt.plot(time, velocities[:, 1], label='v')
+    plt.plot(time, velocities[:, 2], label='w')
+    plt.plot(time, TAS, label='TAS')
+    plt.xlabel('time (s)')
+    plt.ylabel('velocity (m/s)')
+    plt.legend()
+
+    plt.figure('ang velocities')
```

Comment:
@olrosales mentioned that in #58 I don't know if there have been any further progress that we don't see here. Anyway good remark, I completely agree